src/utils/request.ts

```diff
@@ -59,6 +81,45 @@ export const request = (
     });
 }
 
+export const getJwt = (
+    authenticationEndpoint: string,
+    jwtRequestOptions: JwtRequestOptions
+): Promise<JwtResponse> => {
+    return new Promise((resolve, reject) => {
+        var xhr = new XMLHttpRequest();
+        xhr.timeout = 60000;
+        var urlObj = new URL(authenticationEndpoint);
+        urlObj.searchParams.set("t", Math.random().toString());
+        xhr.open('POST', urlObj.toString());
```

Comment:
Directly use the `authenticationEndpoint` string here

Reply:
dn

Reply:
Resolved